src/widgets/Catalog/filters/styles.ts

```diff
@@ -0,0 +1,8 @@
+import { styled } from 'styled-components';
+
+export const WrapFilters = styled.div`
```

Comment:
Название Styled Component начинаем с St
```suggestion
export const StWrapFilters = styled.div`
```